src/main/java/org/apache/accumulo/examples/client/SequentialBatchWriter.java

```diff
@@ -52,14 +49,42 @@ public static Value createValue(long rowId) {
     return new Value(value);
   }
 
+  static class Opts extends ClientOpts {
```

Comment:
Is the example code using these parameters?

Reply:
Yes, it is.  Batch table is created with 10000 rows.

Reply:
I think that is because the code has 10,000 as a literal in the for loop.  Could make the for loop later in the code use `opts.num` instead.  Could make the code use `opts.tableName` instead of the literal `"batch"`

Reply:
Found that --start , --size, --batchMemory, --batchLatency, --batchThreads when removed aren't affecting the test results.  Unsure if there is extra code needed in SequentialBatchWriter or these are unneeded with BasicCompactionStrategy, but were needed for TwoTierCompactionStrategy?

Reply:
I made a commit on GH's UI, not sure if it will compile.  I tried to do a PR, but that failed.  If you do a git pull can you see my changes?

Reply:
Thank you for your help and reviews. I pushed updated files.